WaterSkiBaan/Kabelbaan/LijnenInGebruik.cs

```diff
@@ -8,6 +8,20 @@ namespace WaterSkiBaan.Kabelbaan
 {
     class LijnenInGebruik
     {
-        private LinkedList<Lijn> Lijnen;
+        private LinkedList<Lijn> Lijnen = new LinkedList<Lijn>();
+        private Queue<Lijn> LijnenUitgerangeerd = new Queue<Lijn>();
+
+
+        public void NeemLijnInGebruik(Lijn lijn)
+        {
+            Lijnen.AddLast(lijn);
```

Comment:
"Zorg ervoor dat in de methode de meegegeven lijn wordt toegevoegd aan het begin van de rij."

Hmmm kan het mis hebben, maar moet je niet iets van AddFirst doen?

Reply:
Ja zat hier zelf ook mee. Maar volgens mij begint een rij aan het einde (de eerste in de rij staat aan het einde van de rij) (of je vraag 'waar begint de rij?'). Als je aansluit in een rij krijg je niet nummer 1 maar het hoogste nummer. Met last sluit je achteraan aan ipv vooraan met first. Dit waren mijn gedachten hierover :). Weet niet Hoe @saakee  hier tegenaan kijkt?